terraform/main.tf

```diff
@@ -343,15 +343,6 @@ output "gke_kubeconfig" {
   value = "Run this command to update your kubectl config: gcloud container clusters get-credentials ${module.gke.cluster_name} --region ${var.gcp_region} --project ${var.gcp_project}"
 }
 
-output "specific_manifests" {
```

Comment:
I suspect you can also delete some of the outputs in the `data_share_processor` module that are only used to construct this output